package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "proxy": "http://localhost:8080",
   "dependencies": {
-    "@protonprotocol/proton-web-sdk": "2.6.0",
+    "@proton/web-sdk": "^2.7.13",
```

Comment:
Let's pin the `2.7.13` version so that it only uses the explicitly defined version, instead of automatically updating to the latest minor or patch version.

https://michaelsoolee.com/npm-package-tilde-caret/

We should check whether Taskly's updated `web-sdk` package needs to be pinned as well!